apps/schools/models/assessments.py

```diff
@@ -26,11 +26,11 @@ class Assessment(models.Model):
     start_date = models.DateField(max_length=20,
                                  default=datetime.date.today)
     end_date = models.DateField(max_length=20, default=default_end_date)
-    query = models.CharField(max_length=500, blank=True, null=True)
-    active = models.IntegerField(blank=True, null=True, default=2)
+    query = models.CharField(max_length=500, blank=True, null=True) #Shivangi, is this needed?
+    active = models.IntegerField(blank=True, null=True, default=2) #Shivangi, should this be null or blank?
     double_entry = models.BooleanField('Requires double entry',
             default=True)
-    institutions = models.ManyToManyField(
+    institutions = models.ManyToManyField( #Shivangi, why is this needed?
         'Institution',
```

Comment:
This is not a DB level field. In the database, we have an `AssessmentInstitutionAssociation` table. However, this `institutions` field on the `Assessment` model helps us to run queries easily using the ORM. For example, to get all the institutions related to one Assessment, we can do something like `assessment_instance.institutions.all()`.